hat/assets/js/apps/Iaso/domains/orgUnits/components/TableList.tsx

```diff
@@ -123,7 +138,7 @@ export const TableList: FunctionComponent<Props> = ({
                 saveMulti={saveMulti}
             />
             <Box mt={-4}>
-                <TableWithDeepLink
+                <Table
```

Comment:
in fact i had to use a dedicated query key to trigger filtering only while clicking on search and not while adding a new search. So using params is not possible. I used this because it was already present, it can be removed from the table for sure but we need to keep it for the query key. 
Also i moved from `TableWithDeepLink` to `Table` to be able to use `onTableParamsChange` and ordering, sorting, page without having to use a `useEffect` on parent